### DISCUSSION TITLE
config: setting field using file

### DISCUSSION THREAD

**Discussion by salmanisd** [dexidp/dex#2369] [Q&A] [answered]
As far as I see, we can currently set some fields e.g) ldap bindPW using env variables but currently it's not possible using a secret/pw file?

Is it possible to add such option? (I can gladly take this up).
If yes how should it look like in the config file? something like "passwordFromFile:" ?

**Comment by sagikazarmark** [comment, marked as answer]
The current strategy is to use templating in the config file. See the Dockerfile and the helm chart for more details.
